pkg/cmd/grafana-cli/commands/ls_command.go

```diff
@@ -31,12 +32,12 @@ var validateLsCommand = func(pluginDir string) error {
 }
 
 func (cmd Command) lsCommand(c utils.CommandLine) error {
-	pluginDir := c.PluginDirectory()
-	if err := validateLsCommand(pluginDir); err != nil {
+	pluginsDir := setting.PluginsPath
```

Comment:
Where does `setting.PluginsPath` get set to the same value as the `--pluginsDir` option to grafana-cli?